src/ray/common/ray_config_def.h

```diff
@@ -246,7 +246,8 @@ RAY_CONFIG(int32_t, maximum_profile_table_rows_count, 10 * 1000)
 /// When getting objects from object store, max number of ids to print in the warning
 /// message.
 RAY_CONFIG(uint32_t, object_store_get_max_ids_to_print_in_warning, 20)
-
+/// Number of threads for the GCS internal kv service.
+RAY_CONFIG(uint32_t, gcs_internal_kv_thread_num, 4);
```

Comment:
I not suspect 1 might be sufficient to achieve the same performance? Do you mind experimenting it?

Reply:
I think it should be ok given that in start up, it's actually just 30k kv requests not a big deal actually.

Reply:
oh I feel like you might misinterpret what I said (and I had typo lol). I meant wouldn't having 1 thread enough to achieve the same performance? (or you are saying you prefer to keep it 4?)

Reply:
i mean 1 should be enough and I'm running it. (30k request is not a big deal => 1 thread might be enough) :p

Reply:
yep, they are similar